packages/gauges/src/constants/config/prod.ts

```diff
@@ -1,5404 +1,54 @@
-    token1Address: bscTokens.mgp.address,
-  },
-  {
-    gid: 537,
-    pairName: 'cbBTC-WBTC',
-    address: '0x3E253Ae716eBf5047d6deeE8f2607121ceA08482',
-    chainId: ChainId.ETHEREUM,
-    type: GaugeType.V3,
-    token0Address: ethereumTokens.wbtc.address,
-    token1Address: ethereumTokens.cbBTC.address,
-    feeTier: FeeAmount.LOWEST,
-  },
-  {
-    gid: 538,
-    pairName: 'cbBTC-ETH',
-    address: '0xF84AAd7ab687b1BB1FBB190344240b60B39e5b79',
-    chainId: ChainId.ETHEREUM,
-    type: GaugeType.V3,
-    token0Address: ethereumTokens.weth.address,
-    token1Address: ethereumTokens.cbBTC.address,
-    feeTier: FeeAmount.LOW,
-  },
-  {
-    gid: 539,
-    pairName: 'cbBTC-USDC',
-    address: '0x5fbbC3380DD9f7253Aa10838DD641dAc8242F817',
-    chainId: ChainId.ETHEREUM,
-    type: GaugeType.V3,
-    token0Address: ethereumTokens.usdc.address,
-    token1Address: ethereumTokens.cbBTC.address,
-    feeTier: FeeAmount.LOW,
-  },
-  {
-    gid: 540,
-    pairName: 'cbBTC-ETH',
-    address: '0xC211e1f853A898Bd1302385CCdE55f33a8C4B3f3',
-    chainId: ChainId.BASE,
-    type: GaugeType.V3,
-    token0Address: baseTokens.weth.address,
-    token1Address: baseTokens.cbBTC.address,
-    feeTier: FeeAmount.LOWEST,
-  },
-  {
-    gid: 541,
-    pairName: 'cbBTC-USDC',
-    address: '0xb94b22332ABf5f89877A14Cc88f2aBC48c34B3Df',
-    chainId: ChainId.BASE,
-    type: GaugeType.V3,
-    token0Address: baseTokens.usdc.address,
-    token1Address: baseTokens.cbBTC.address,
-    feeTier: FeeAmount.LOWEST,
-  },
-  {
-    gid: 542,
-    pairName: 'cbBTC-USDbC',
-    address: '0x55B40f8911402568414ec6e7e0d2D87c3D19e65C',
-    chainId: ChainId.BASE,
-    type: GaugeType.V3,
-    token0Address: baseTokens.cbBTC.address,
-    token1Address: baseTokens.usdbc.address,
-    feeTier: FeeAmount.LOWEST,
-  },
-  {
-    gid: 543,
-    pairName: 'LAVA-ETH',
-    address: '0xB593f062D5F235862643fF0DBAb743d7316B5b9F',
-    chainId: ChainId.BASE,
-    type: GaugeType.V3,
-    token0Address: baseTokens.lava.address,
-    token1Address: baseTokens.weth.address,
-    feeTier: FeeAmount.MEDIUM,
-  },
-  {
-    gid: 544,
-    pairName: 'EURC-USDC',
-    address: '0x1Ca42C7219F0cB1B67927e26502320cB98F725bd',
-    chainId: ChainId.BASE,
-    type: GaugeType.V3,
-    token0Address: baseTokens.eurc.address,
-    token1Address: baseTokens.usdc.address,
-    feeTier: FeeAmount.LOWEST,
-  },
-  {
-    gid: 545,
-    pairName: 'EURC-ETH',
-    address: '0xb57BdfDb5FE848f999B45A868131a50b2EBE1755',
-    chainId: ChainId.BASE,
-    type: GaugeType.V3,
-    token0Address: baseTokens.weth.address,
-    token1Address: baseTokens.eurc.address,
-    feeTier: FeeAmount.LOWEST,
-  },
-]
+import retry from 'async-retry'
+import { GaugeConfig } from '../../types'
+
+let cache: GaugeConfig[] | undefined
+let promiseQueue: ((value: GaugeConfig[] | PromiseLike<GaugeConfig[]>) => void)[] = []
+let inProgress = false
+
+export const CONFIG_PROD = async (): Promise<GaugeConfig[]> => {
```

Comment:
Can we rename and move this function somewhere else? It doesn't make sense to call it `CONFIG_PROD`. Maybe `queries/getGauges.ts`